weave-js/src/components/Panel2/PanelFacet/Component.tsx

```diff
@@ -326,109 +325,101 @@ const PanelFacetGridMode: React.FC<PanelFacetProps> = props => {
+      )}
+      {Object.keys(xPos).map(xKey => (
+        <S.FacetHeaderCell
+          key={'col-' + xKey}
+          style={{height: '24px'}}
+          gridColumnStart={xPos[xKey] + columnOffset + 1}
+          gridRowStart={rowOffset}>
+          {xKey}
+        </S.FacetHeaderCell>
+      ))}
+      {Object.keys(yPos).map(yKey => (
+        <S.FacetHeaderCell
+          key={'row-' + yKey}
+          gridColumnStart={columnOffset}
+          gridRowStart={yPos[yKey] + rowOffset + 1}>
+          {yKey}
+        </S.FacetHeaderCell>
+      ))}
+      {cellNodes.map((cellNode, i) => {
+        const groupKey = groupKeys[i];
+
+        if (groupKey == null) {
+          // adding the key gets rid of the key warning
+          return <div key={'null-cell-' + i}>-</div>;
+        }
+
+        const xKey = groupKey[xColName];
+        const yKey = groupKey[yColName];
+        const selected =
+          xKey === config.selectedCell?.x && config.selectedCell?.y === yKey;
+        return (
           <div
-            key={'row-' + yKey}
+            key={'cell-' + xPos[xKey] + '-' + yPos[yKey]}
             style={{
-              gridRowStart: yPos[yKey] + 2,
-              gridColumnStart: 1,
-              overflow: 'hidden',
-              whiteSpace: 'nowrap',
-              textOverflow: 'ellipsis',
-              fontSize: 12,
-            }}>
-            {yKey}
+              gridColumnStart: xPos[xKey] + columnOffset + 1,
+              gridRowStart: yPos[yKey] + rowOffset + 1,
+              border: selected ? `2px solid ${MOON_800}` : undefined,
+            }}
+            onClick={() =>
+              props.updateConfig({selectedCell: {x: xKey, y: yKey}})
+            }>
+            <PanelContextProvider newVars={cellVars}>
+              <SelectPanel
+                inputNode={cellNode}
+                selectFunction={cellFunction}
+                vars={table.columns[dims.select].panelVars ?? {}}
+                panelId={table.columns[dims.select].panelId}
+                config={table.columns[dims.select].panelConfig}
+                panelContext={props.context}
+                updateConfig={newConfig =>
+                  props.updateConfig({
+                    table: TableState.updateColumnPanelConfig(
+                      table,
+                      dims.select,
+                      newConfig
+                    ),
+                  })
+                }
+                updatePanelContext={props.updateContext}
+              />
+            </PanelContextProvider>
           </div>
-        ))}
-        {cellNodes.map((cellNode, i) => {
-          const groupKey = groupKeys[i];
-
-          if (groupKey == null) {
-            return <div>-</div>;
-          }
-
-          const xKey = groupKey[xColName];
-          const yKey = groupKey[yColName];
-          const selected =
-            xKey === config.selectedCell?.x && config.selectedCell?.y === yKey;
-          return (
-            <Resizable
```

Comment:
Was dropping the resize intentional?

Reply:
yes resize is now intended to scale directly with the overall panel size